portfolio/src/main/webapp/scripts/snake/sketch.js

```diff
@@ -1,5 +1,5 @@
 /** Game board */
-let cnv;
+let canvas;
 /** Game board container in HTML */
 let cnvDiv;
```

Comment:
Nit: Sorry I meant to suggest as well that this could be 'canvasDiv' :)

Reply:
Gotcha, pushed the change!